pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2445,6 +2512,21 @@
       (handle-response http-event)
     [moves http-server-gate]
   ::
+  ++  watch-cache
+    ::
```

Comment:
Unnecessary comment

Reply:
I don't get it either, but it seems to be the convention in this file